third_party/silabs/SiWx917_sdk.gni

```diff
@@ -46,13 +46,13 @@ template("siwx917_sdk") {
     _include_dirs = [
       "${chip_root}",
       "${chip_root}/examples/platform/silabs/SiWx917/SiWx917",
-      "${sdk_support_root}/matter/si91x/siwx917/BRD4325x/support/hal",
+      "${sdk_support_root}/matter/si91x/siwx917/${sdk_support_board}/support/hal",
```

Comment:
To logic for setting `sdk_support_board` could have been done directly in this file and just have this variable as local to `SiWx917_sdk.gni`. But I guess that work also. 
Do we anticipate more boards that would be sharing generated bringup files?

Reply:
As of now, no more boards will be sharing the generated files.
To avoid duplication we will be using BRD4325x for the boards BRD4325B/C/G

Reply:
Then if that is the case, It would be preferable to keep all this logic in SiWx917_sdk.gni and move out the additions in `silabs_boards.gni`

Reply:
Sure